content/en/docs/ops/configuration/telemetry/telemetry-v1-vs-v2/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Differences between Telemetry v1 and v2
+description: Tradeoffs and gaps between Telemetry v1 and v2.
+weight: 10
+owner: istio/wg-policies-and-telemetry-maintainers
+test: no
+---
+
```

Comment:
Can we provide a summary statement here?  Something like: `Istio has transitioned to a new platform for extensibility and telemetry generation. While this new platform unlocks several critical use cases and provides numerous performance benefits, there are some features of the prior platform that are not supported as of Istio 1.7. The Istio community is working hard to close the remaining gaps, which are discussed below.` ?